src/runUtils.ts

```diff
@@ -91,7 +93,33 @@ export function getHostOutputDir(): string {
     : getFilesDir();
 }
 
+const DEFAULT_FILES_DIR = '/files';
```

Comment:
Given that we want to not mount the volume (and therefore not have persistent storage) if I don't define the `FILES_DIR` var, I'd suggest to remove all the code related to `DEFAULT_FILES_DIR`, as it introduces a side effect that the user doesn't expect